lib/blockchain/cardano-build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+if [ -z "$USER" ]; then
+    USER=`whoami`
+fi
+
+tmpdir=$(mktemp -d -t nix-XXXXXXXXXX)
+cd $tmpdir
+wget -q https://nixos.org/nix/install
+chmod +x install
+OUTPUT=$(./install --no-daemon)
```

Comment:
why do we need `OUTPUT` ?

Reply:
old thing when i parsed result of installation, can be removed